python/cog/server/runner.py

```diff
@@ -316,7 +316,16 @@ def _predict(
 
     for k, v in input_dict.items():
         if isinstance(v, types.URLPath):
-            input_dict[k] = v.convert()
+            try:
+                input_dict[k] = v.convert()
+            except Exception as e:
+                # Failed to get remote file.
+                # We do not re-raise since it is not an inference error.
```

Comment:
We don't reraise for inference errors either, only for irrecoverable errors, so I think we can drop this comment.

Reply:
Ah... Fair, got it!